### DISCUSSION TITLE
C1CountryCode

### DISCUSSION THREAD

**Discussion by machnst1** [OxalisCommunity/oxalis#649] [Q&A] [answered]
If i use Oxalis packages for creating the Header it works fine even if COUNTRY_C1 is missing from the sbdh
But if i then try to send the document with a Header that does not have the c1CountryCodeIdentifier it fails.
Is it a way to set the Header.c1CountryCode to for example ZZ now up until 1st of JAN

**Comment by OysteinLq** [comment, marked as answer]
You may want to run Oxalis 6.1.1, which is the only version that supports C1 but doesn't yet make it mandatory. We've had no issues sending or receiving documents both with and without C1 using that.

(Note that you'll then have to use the workaround described in https://github.com/OxalisCommunity/Oxalis-AS4/issues/217#issuecomment-1750512261 )
